src/app/role-admin/employee-list/employee-list.component.css

```diff
@@ -0,0 +1,387 @@
+/* 
+*{
+    box-sizing: border-box;
+    overflow: hidden;
+} */
+
+
+.whole {
+    background-color: #abeafd;
+    /* width: 740px; */
+    padding-top: 10px;
+    padding-right: 20px;
+    margin-left:2px;
+    height: auto;
+    /* overflow-y: scroll; */
+    @media screen and (max-width:600px){
+        background-color:black;
+        font-size:50px
+        
+    }
+
+}
+.scroll{
+    overflow-y: scroll ;
+
+}
+
+.emp-list {
+    width: 700px;
+    margin-left: 18px;
+    /* background-color: whitesmoke; */
+    padding-bottom: 10px;
+    height: 1000px;
+
+}
+
+.searchingarea {
```

Comment:
naming convention either use snake case or hyphen. for searching area you can have a meaningful example: search box
do same for all other places